client/app/pages/tournament-settings/tournament-settings.component.ts

```diff
@@ -66,19 +80,42 @@ export class TournamentSettingsComponent implements OnInit {
         this.tournamentSettingsService.saveSetting(setting, this.settings[setting].value).subscribe(
             response => {
                 this.loading = false;
-                this.notification.success("Setting saved successfully")
+                this.notification.success(`${this.settings[setting].display} saved successfully`)
             },
             err => {
                 this.notification.error("Oh no! Something went wrong while trying to save...")
             })
     }
 
-    setDeleteModel(model){
+    saveNumberOFRounds() {
+        if (this.doesValueBetween(this.settings['numberOfPracticeRounds'].value, this.MIN_AMOUNT_OF_ROUNDS, this.MAX_AMOUNT_OF_ROUNDS) &&
+            this.doesValueBetween(this.settings['numberOfRankingRounds'].value, this.MIN_AMOUNT_OF_ROUNDS, this.MAX_AMOUNT_OF_ROUNDS)) {
+            forkJoin(this.tournamentSettingsService.saveSetting('numberOfPracticeRounds', this.settings['numberOfPracticeRounds'].value),
+                     this.tournamentSettingsService.saveSetting('numberOfRankingRounds', this.settings['numberOfRankingRounds'].value))
+                .subscribe(
+                    response => {
+                        this.notification.success(`Number of rounds saved successfully`)
+                    },
+                    error => {
+                        this.notification.error("Oh no! Something went wrong while trying to save number of rounds")
+                    }
+                )
+        } else {
+            this.notification.error(`Number of round per stage need to be between ${this.MIN_AMOUNT_OF_ROUNDS} and ${this.MAX_AMOUNT_OF_ROUNDS}`)
```

Comment:
Replace "Number of round per stage need to be between" with
"Valid rounds per stage is between"